Libraries/BatchedBridge/MessageQueue.js

```diff
@@ -310,10 +318,9 @@ class MessageQueue {
 
   __callImmediates() {
     Systrace.beginEvent('JSTimers.callImmediates()');
-    if (!JSTimers) {
-      JSTimers = require('JSTimers');
+    if (this._immediatesCallback) {
```

Comment:
For explicitness, can you change this check to `this._immediatesCallback != null`?